src/Npgsql.GeoJSON/Npgsql.GeoJSON.csproj

```diff
@@ -4,8 +4,7 @@
     <Description>GeoJSON plugin for Npgsql, allowing mapping of PostGIS geometry types to GeoJSON types.</Description>
     <PackageTags>npgsql postgresql postgres postgis geojson spatial ado ado.net database sql</PackageTags>
     <VersionPrefix>1.0.0</VersionPrefix>
-    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT' OR '$(CoreOnly)' == 'True'">netstandard2.0</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I assume you mean to change all these to `<TargetFrameworks>netstandard2.0;net461</TargetFrameworks>`

Reply:
Not really - there's no actual reason to multitarget for these particular projects (Npgsql plugins as opposed to Npgsql itself). See the first commit's comment for some background.

Reply:
Ah, didn't read the PR comment. A lot of the time it's still a good idea to target `net461` if you intend to support it, I wrote a thing a while back:
https://stu.dev/dotnet-framework-support-for-dotnetnet-standard-2-0/

Reply:
I was somewhat aware of these issues but not that it was a real problem. I'll revert this change as it doesn't really win us anything, thanks!